src/main/java/net/fabricmc/fabric/impl/registry/RemappableRegistry.java

```diff
@@ -20,6 +20,12 @@
 import net.minecraft.util.Identifier;
 
 public interface RemappableRegistry {
-	void remap(Object2IntMap<Identifier> idMap, boolean reallocateMissingEntries) throws RemapException;
+	enum RemapMode {
+		AUTHORITATIVE, /* remapping the server; any changes allowed */
```

Comment:
These should be javadocs, not comment blocks

Reply:
This is internal code for the time being